mongodbforms/documents.py

```diff
@@ -357,7 +358,7 @@ def __new__(cls, name, bases, attrs):
         return new_class
 
 
-class BaseDocumentForm(BaseForm):
+class BaseDocumentForm(BaseModelForm):
```

Comment:
unit tests are not enough. If we roll this out to work4us it might break the Django admin, very hard.